cicd/cji_smoke_test.sh

```diff
@@ -0,0 +1,52 @@
+# Run smoke tests as a ClowdJobInvocation deployed by bonfire
+
+# Env vars defined by caller:
+#IQE_PLUGINS="plugin1,plugin2" -- pytest plugins to run separated by "," #IQE_MARKER_EXPRESSION="mymarker" -- pytest marker expression
+#IQE_FILTER_EXPRESSION="something AND something_else" -- pytest filter, can be "" if no filter desired
+#NAMESPACE="mynamespace" -- namespace to deploy iqe pod into, can be set by 'deploy_ephemeral_env.sh'
+
+CJI_NAME="$APP_NAME-smoke-tests"
+
+function kill_port_fwd {
+    echo "Caught signal, kill port forward"
+    if [ ! -z "$PORT_FORWARD_PID" ]; then kill $PORT_FORWARD_PID; fi
+}
+
+# Invoke the CJI with the exported vars from pr_check
+pod=$(bonfire deploy-iqe-cji $APP_NAME -m $IQE_MARKER_EXPRESSION -e "clowder_smoke" --cji-name $CJI_NAME -n $NAMESPACE)
```

Comment:
We'll also want to pass `-k $IQE_FILTER_EXPRESSION`

marker or filter could potentially be undefined... I think if you default them here in the bash script to `'""'` if undefined it should work?